asia-pacific-energy-research-centre/8th-Outlook
# Updated EGEDA data

The previous EGEDA data was not actually in 'tidy' format! I modified the script so to correct that. I also added in a conversion for ktoe to PJ. Values in both units are written to the csv file.

The demand model notebooks will need to be modified slightly to use the updated EGEDA data. I made the modifications in the Agriculture Demand notebook. 

Demand modelers, please test the notebooks.

To test:
1. Run the EGEDA notebook.
2. Run the Agriculture notebook to check that it works. 

Once everyone is comfortable with the changes we can merge them in to the master branch.

**Note: you will need to run the EGEDA notebook to make the new data file because it is too big to commit.**
